qualtran/bloqs/util_bloqs.py

```diff
@@ -471,3 +471,41 @@ def on_classical_vals(self, reg: int) -> Dict[str, 'ClassicalValT']:
 
     def _t_complexity_(self) -> 'TComplexity':
         return TComplexity()
+
+
+@frozen
+class Power(Bloq):
```

Comment:
is this addition on purpose? factor into its own PR? is it used elsewhere in the control stuff?

Reply:
Yes, this addition is on purpose and was required to pass tests (eg: for phase estimation, where we use powers of controlled gate). I can factor into it's own PR but this PR will be blocked on the other PR being merged. I also need to add tests for this bloq

Reply:
no need to factor